plugins/state_monitor.py

```diff
@@ -20,6 +21,8 @@ def __init__(
         low_space=100,
         slack_channel=None,
         slack_key_file=None,
+        google_chat_channel=None,
+        google_chat_key_file=None,
```

Comment:
```suggestion
        channel=None,
        key_file=None,
```

Assuming you go with the suggestion above